charts/reports-server/templates/deployment.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       labels:
-        {{- include "reports-server.selectorLabels" . | nindent 8 }}
+        {{- include "reports-server.labels" . | nindent 8 }}
```

Comment:
why remove the selectorLabels?

Reply:
because they are included in reports-server.labels